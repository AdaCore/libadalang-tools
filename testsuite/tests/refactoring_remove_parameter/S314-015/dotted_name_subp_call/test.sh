#!/bin/sh
remove_parameter -P default.gpr -S my_package.ads -L 3 -R 32
