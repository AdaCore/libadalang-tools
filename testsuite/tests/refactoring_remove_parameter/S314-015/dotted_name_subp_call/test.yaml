driver: shell_script

description: refactoring remove parameter tool test
