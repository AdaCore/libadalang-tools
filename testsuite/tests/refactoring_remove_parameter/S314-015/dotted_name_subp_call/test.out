Removing parameter B1
main.adb
5:33-5:40 
6:49-6:62 
my_package.adb
3:30-3:34 
my_package.ads
3:30-3:34 

